### ISSUE TITLE
Check if true

### ISSUE THREAD

**Issue by BorghildSelle** [equinor/energyvision#2351]
From Tailwind:
Unstyled lists are not announced as lists by VoiceOver If your content is truly a list but you would like to keep it unstyled, add a “list” role to the element:

Check if this is true with our promotion lists, cards list, events lists, promotiles etc.

**Comment by padms** [comment]
Events , Promotiles are not lists but divs

**Comment by BorghildSelle** [comment]
they are getting fixed in branch bose/2341
